### PR TITLE
Updating UX colors

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: src/app/public/src/modules/tutorial/tutorial-summary.component.scss [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
 
-  @media (min-width: $stache-screen-sm) {
+  @media (min-width: $stache-screen-sm-min) {
     font-size: 19.5px;
   }
 
   &,
   p {
-    color: $stache-color-gray;
+    color: $stache-text-color-deemphasized;
```
Comment (Blackbaud-BrandonJones) [inline src/app/public/src/modules/tutorial/tutorial-summary.component.scss]: This color isn't enough of a contrast with the well.   fails on the http://webaim.org/resources/contrastchecker/

`#686c73` - font color
`#eeeeef` - background color

File: src/app/public/src/modules/back-to-top/back-to-top.component.scss [hunk 2]
```diff
@@ -15,9 +15,8 @@
   opacity: 1;
   transition: opacity .5s;
 
-  &:hover, 
+  &:hover,
```
Comment (Blackbaud-BrandonJones) [inline src/app/public/src/modules/back-to-top/back-to-top.component.scss]: Why did we remove the color change on hover?

File: src/app/public/src/modules/tutorial/tutorial-summary.component.scss (hunk 1 above)
Comment (Blackbaud-BrandonJones) [inline src/app/public/src/modules/tutorial/tutorial-summary.component.scss]: EDIT-   It Fails the AAA check for normal font.   passes the AA.  Not sure what standard we are aiming for here, but I figured AAA was the goal.

File: src/app/public/src/modules/code/code.component.scss [hunk 3]
```diff
@@ -3,9 +3,9 @@
 .stache-code {
   font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
   font-size: 90%;
-  background-color: scale-color($color: $stache-color-red, $lightness: 90%);
+  background-color: $stache-background-color-selected;
   border-radius: $stache-border-radius;
-  border: 1px solid scale-color($color: $stache-color-red, $lightness: 70%);
-  color: scale-color($color: $stache-color-red, $lightness: -20%);
+  border: $stache-border-color-neutral-medium;
```
Comment (Blackbaud-BrandonJones) [inline src/app/public/src/modules/code/code.component.scss]: need 1px solid here.

Comment (Blackbaud-SteveBrush) [inline src/app/public/src/modules/code/code.component.scss]: I'm leaning toward removing the border altogether...

**Comment by Blackbaud-BrandonJones** [comment]
LGTM

**Pull request merged**